crates/sui-indexer/migrations/pg/2023-10-06-204335_tx_indices/up.sql

```diff
@@ -54,14 +62,24 @@ CREATE TABLE tx_calls_fun (
     PRIMARY KEY(package, module, func, tx_sequence_number)
 );
 CREATE INDEX tx_calls_fun_sender ON tx_calls_fun (sender, package, module, func, tx_sequence_number);
+CREATE INDEX tx_calls_fun_tx_sequence_number ON tx_calls_fun (tx_sequence_number);
 
 CREATE TABLE tx_digests (
     tx_digest                   BYTEA        PRIMARY KEY,
     tx_sequence_number          BIGINT       NOT NULL
 );
+CREATE INDEX tx_digests_tx_sequence_number ON tx_digests (tx_sequence_number);
 
 CREATE TABLE tx_kinds (
     tx_sequence_number          BIGINT       NOT NULL,
     tx_kind                     SMALLINT     NOT NULL,
     PRIMARY KEY(tx_kind, tx_sequence_number)
 );
+CREATE INDEX tx_kinds_tx_sequence_number ON tx_kinds (tx_sequence_number);
+
+CREATE TABLE cp_tx (
```

Comment:
Looks like this table is mainly used as watermarks for the pruner, not just a mapping between cp and tx. Should the name reflect this purpose better?